src/selector.ts

```diff
@@ -56,7 +56,7 @@ class OpProp {
     for (let i = 0; i < raw.length; i += 1) {
       const codePoint = raw.charCodeAt(i);
       // the codePoint appears to be some form of comparison operator we support
-      if (codePoint === 33 || (codePoint >= 60 && codePoint <= 62)) {
+      if (codePoint === 33 || (codePoint >= 60 && codePoint <= 62) || codePoint === 94) {
```

Comment:
Nit: It would be nice to put inline comments for which code point is which character.